tests/providers/jersey-tides.test.ts

```
@@ -0,0 +1,262 @@
+}));
+
+import fetch from 'node-fetch';
+import { writeFileSync } from 'node:fs';
+
+const mockFetch = fetch as any;
+const mockWriteFileSync = writeFileSync as any;
+
+beforeEach(() => {
+  vi.clearAllMocks();
+  // Clear environment variables
+  delete process.env.STORM_TOKEN;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)